docs/metadata.rst

```diff
@@ -130,26 +258,27 @@ Examples
 As an example here is a schema using the ``struct`` codec which could apply, for example,
 to the individuals in a tree sequence:
 
-.. code-block:: json
-
-    {
-      "codec": "json",
-      "type": "object",
-      "properties": {
-        "accession_number": {"type": "number", "binaryFormat": "i"},
-        "collection_date": {
-          "name": "Collection date",
-          "description": "Date of sample collection in ISO format",
-          "type": "string",
-          "binaryFormat": "10p",
-          "pattern": "^([1-9][0-9]{3})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])?$"
-        },
-      },
-      "required": ["accession_number"],
-      "additionalProperties": false,
-    }
-
-This schema states that the metadata for the each row of the table
+.. code-block:: python
+
+    schema = metadata.MetadataSchema(
+        {
+            "codec": "struct",
+            "type": "object",
+            "properties": {
+                "accession_number": {"type": "number", "binaryFormat": "i"},
+                "collection_date": {
+                    "name": "Collection date",
+                    "description": "Date of sample collection in ISO format",
+                    "type": "string",
+                    "binaryFormat": "10p",
+                    "pattern": "^([1-9][0-9]{3})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])?$",
+                },
+            },
+            "additionalProperties": False,
+        }
+    )
+
+This schema states that the metadata for each row of the table
 is an object consisting of two properties. Property ``accession_number`` is a number
 (stored as a 4-byte int) which must be specified (it is included in the ``required``
```

Comment:
they're both required, since it's `struct`, and there isn't a `required` list any more (should there be, to emphasize?)

Reply:
Fixed in https://github.com/tskit-dev/tskit/pull/741/commits/4dfd4810fcb7a4896378b8259d43d62d17869fe6